### PR TITLE
MSPSDS-220 create staging

### PR DESCRIPTION
**Pull request by AnthonyPengelly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,12 @@ This project is following the GOV UK style guides.
 We have used the GOV UK elements library to define CSS classes.
 The design guide for this is [here](http://govuk-elements.herokuapp.com/).
 
+## Deployment
+Anything which is pushed to `master` will trigger the [travis build](https://travis-ci.org/UKGovernmentBEIS/beis-mspsds) and cause a deployment to the INT site.
+
+Anthing pushed to the branch `staging` will cause travis to instead build to the STAGING environment. Please only do this if you are confident that this is
```
Comment (broder) [inline README.md]: We should lock the `staging` branch such that it needs a PR to be updated

Comment (AnthonyPengelly) [reply README.md]: Good plan! @samlionheart, do you have admin permissions on this repo?

Comment (samlionheart) [reply README.md]: @AnthonyPengelly I've made staging require pr review before merging, is that okay?

Comment (AnthonyPengelly) [reply README.md]: Sounds good to me! thanks :)

**Pull request merged**